manifests/appcontroller.yaml

```diff
@@ -3,10 +3,10 @@ kind: Pod
 metadata:
   name: k8s-appcontroller
   annotations:
-    pod.alpha.kubernetes.io/init-containers: '[{"name": "kubeac-bootstrap", "image": "mirantis/k8s-appcontroller", "command": ["bootstrap", "/opt/kubeac/manifests"]}]'
+    pod.alpha.kubernetes.io/init-containers: '[{"name": "kubeac-bootstrap", "image": "mirantis/k8s-appcontroller:service-state", "command": ["bootstrap", "/opt/kubeac/manifests"]}]'
```

Comment:
Yes, I forgot about this, good catch. Indeed, after reviewing I will need to update this.